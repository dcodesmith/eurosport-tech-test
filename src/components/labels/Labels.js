import React from 'react';

const Labels = () => (
  <div className="column">
    <div className="player">
      <ul>
        <li>Rank</li>
        <li>Age</li>
        <li>Country</li>
        <li>Points</li>
        <li>Height</li>
        <li>Weight</li>
        <li>Wins</li>
        <li>Losses</li>
      </ul>
    </div>
  </div>
);

export default Labels;