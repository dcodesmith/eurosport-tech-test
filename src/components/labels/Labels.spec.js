import React from 'react';
import { shallow } from 'enzyme';

import Labels from './';

describe('Given a Labels Component', () => {
	describe('When rendered', () => {
    let LabelsComponent;
      
		beforeAll(() => {
      LabelsComponent = shallow(<Labels/>);
		});

    it('should display the statistics labels', () => {
      expect(LabelsComponent).toMatchSnapshot();
    });
	});
});