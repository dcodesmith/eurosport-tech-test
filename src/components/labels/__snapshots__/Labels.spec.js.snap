// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Labels Component When rendered should display the statistics labels 1`] = `
<div
  className="column"
>
  <div
    className="player"
  >
    <ul>
      <li>
        Rank
      </li>
      <li>
        Age
      </li>
      <li>
        Country
      </li>
      <li>
        Points
      </li>
      <li>
        Height
      </li>
      <li>
        Weight
      </li>
      <li>
        Wins
      </li>
      <li>
        Losses
      </li>
    </ul>
  </div>
</div>
`;
